ci/pipeline.yml

```diff
@@ -616,6 +616,9 @@ jobs:
                 - metarpheus
                 - tapiro
                 - docs
+                - wiro
+                - scalafmt
+                - java-time-circe-codecs
```

Comment:
I think `release` should depend on all these too since it releases all libraries.